packages/caravan-psbt/src/psbtv2/psbtv2.ts

```diff
@@ -915,6 +915,7 @@ export class PsbtV2 extends PsbtV2Maps {
     redeemScript,
     witnessScript,
     bip32Derivation,
+    sighashType,
```

Comment:
While not required on a psbt input, this field was required for constructing a complete, unsigned txn to match the test vectors unsigned raw txn.